health_check/contrib/celery/apps.py

```diff
@@ -8,4 +9,20 @@ class HealthCheckConfig(AppConfig):
 
     def ready(self):
         from .backends import CeleryHealthCheck
-        plugin_dir.register(CeleryHealthCheck)
+
+        # Celery queues defined in SETTINGS as CELERY_QUEUES:
+        for queue, queue_dict in settings.CELERY_QUEUES.items():
```

Comment:
I would recommend to use `current_app.amqp.queues` as this will return all queues even if you only have task routing defined and implicit queues.

Reply:
Done.

Reply:
Hi @mmagician yes, it works in your tests because you set the setting earlier. For it to actually work in your tests you will need to initiate Celery in the test app.

```python
app = Celery('proj')
app.config_from_object('django.conf:settings', namespace='CELERY')
```
See, http://docs.celeryproject.org/en/latest/django/first-steps-with-django.html